frontend/src/components/SyncEditor/SyncEditor.tsx

```diff
@@ -652,12 +653,22 @@ export function SyncEditor(props: SyncEditorProps): JSX.Element {
     if (variant === 'toolbar') {
       height -= 36
     }
+    if (editorHasErrors) {
+      height -= 36
+    }
     editorRef?.current?.layout({ width, height })
     setShowCondensed(width < 500)
   })
 
   return (
     <div ref={pageRef} className="sync-editor__container">
+      {editorHasErrors && (
+        <Alert
+          variant="danger"
+          title={t('Form edits will be ignored until YAML syntax errors are fixed.')}
+          actionClose={<AlertActionCloseButton onClose={() => setEditorHasErrors(false)} />}
```

Comment:
If you dismiss the alert, it actually stops ignoring changes, so your changes will be lost if you touch something in the form. And, if you continue editing without fixing the YAML syntax, the alert re-appears immediately. Should we just remove the close button?

Reply:
yep--I was going to use it as a way to let the user throw away their editor changes but now it's just confusing